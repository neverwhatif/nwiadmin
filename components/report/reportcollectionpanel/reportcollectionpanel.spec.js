import React from 'react';
import { shallow } from 'enzyme';

import ReportCollectionPanel from './index';

describe('ReportCollectionPanel', () => {
    const component = shallow(<ReportCollectionPanel><div /></ReportCollectionPanel>);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
});
