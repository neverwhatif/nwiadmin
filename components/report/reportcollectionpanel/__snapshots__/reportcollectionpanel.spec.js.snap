// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportCollectionPanel should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ReportCollectionPanel
    isDisabled={false}
    title={null}
  >
    <div />
  </ReportCollectionPanel>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="panel"
      >
        <div />
      </div>,
      "className": "root",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          null,
          <div />,
        ],
        "className": "panel",
      },
      "ref": null,
      "rendered": Array [
        null,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="panel"
        >
          <div />
        </div>,
        "className": "root",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            null,
            <div />,
          ],
          "className": "panel",
        },
        "ref": null,
        "rendered": Array [
          null,
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
